app/scripts/directives/util.js

```diff
@@ -177,4 +177,18 @@ angular.module('openshiftConsole')
         }
       });
     };
+  }).directive('linkDisabled', function(){
+    return {
+      restrict: 'A',
+      scope: {
+        linkDisabled: '='
+      },
+      link: function(scope, element) {
+        element.bind('click', function(event) {
```

Comment:
click handlers fire in the order they are bound, its possible this prevention handler could be bound in a different order than the click handler you are trying to prevent from firing